src/pages/iou/steps/MoneyRequestAmount.js

```diff
@@ -167,20 +167,21 @@ const replaceAllDigits = (text, convertFn) =>
         .join('')
         .value();
 
-function MoneyRequestAmountPage(props) {
+function MoneyRequestAmount(props) {
     const {translate, toLocaleDigit, fromLocaleDigit, numberFormat} = useLocalize();
     const selectedAmountAsString = props.iou.amount ? CurrencyUtils.convertToWholeUnit(props.iou.currency, props.iou.amount).toString() : '';
 
     const prevMoneyRequestID = useRef(props.iou.id);
     const textInput = useRef(null);
     const iouType = useRef(lodashGet(props.route, 'params.iouType', ''));
     const reportID = useRef(lodashGet(props.route, 'params.reportID', ''));
-    const isEditing = useRef(lodashGet(props.route, 'path', '').includes('amount'));
+    const isEditing = useRef(Navigation.getActiveRoute().includes('amount'));
```

Comment:
Is this because the TopBar screen does not pass route? IF so can we use the defined `currentRoute` below?

Or can we pass route?

Reply:
`currentRoute` = `{"name":"manual","key":"manual-JmL5yMm-CMpq21hc-sMFc","params":{"reportID":"65212528","iouType":"request"}}`

`Navigation.getActiveRoute()` = `"/request/new/65212528/manual"`

I could look into passing route, but this worked.

Reply:
route seems available already.